src/styles/blocks/style-block-card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
+}
+
+.card--size {
+  box-sizing: border-box;
+  width: 200px;
+
+}
+
+.card__image {
+  display: flex;
+  background-size: cover;
+  height: 134px;
+  width: 160px;
+  margin: 32px auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: hsl(234, 80%, 12%);
+  width: 166px;
+  /* height: 36px; */
+  margin: 0 16px 4px;
+}
+
+.card__subtitle {
+  width: 98px;
+  /* height: 14px; */
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+  margin: 0 0 16px 16px;
+}
+
+.card__prise {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  gap: 60px;
+  width: 166px;
+  margin: 0 auto 16px;
```

Comment:
I think display: flex and justify-content: space-between for container should be enough and you can remove other styles